spec/fixtures/context/basic.y

```diff
@@ -18,6 +18,13 @@
 %printer {
     print_token();
 } tNUMBER tSTRING
+%destructor {
```

Comment:
Is this intentional change or forget to remove after debug?

Reply:
Intentionally change it. I noticed in spec/lrama/context_spec.rb that `.flatten` is needed in `%printer`. So I also added `%destructor` and `%error-token`. However, I am sure that they are not essential tests in this file, so you may want to check them with other tests. What do you think?